code/index.html

```diff
@@ -3,28 +3,232 @@
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
+    <nav>
+      <div class="nav-content">
+        <div class="nav-logo"><img src="assets/hibr-logo.svg"></div>
+        <div class="nav-links">
+          <a href="#" class="nav-link">What's Hibr™?</a>
+          <a href="#" class="nav-link">Features</a>
+          <a href="#" class="nav-link">Hibr™ a friend</a>
+          <a href="#" class="nav-link">Legal</a>
+          <a href="#" class="nav-link">Guestbook</a>
+        </div>
+      </div>
 
-    <!-- Signup form -->
+    </nav>
+    
+    <header>  
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+      <section class="header-content">
+        <img src="assets/hibr-logo.svg" class="logo">
+        <p>Because right now—we could all use a good year’s sleep.</p>
+        <section class="header-cta">
+          <button class="button-primary teal-bg"><p>Sign up</p></button>
+          <button class="button-secondary teal"><p>Learn more</p></button>
+        </section>
+      </section>
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+      <div class="video-container">
+        <video autoplay muted loop>
+            <source src="videos/sheep.mp4" type="video/mp4" />
+        </video>
+        <h2>Your caption here</h2>
       </div>
 
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+    </header>
+
+    <section class="productInfo">
+      <div class="productInfo-content">
+        <div class="hibr-info">
+          <h2>We get it.</h2>
+          <p>What a year, huh? A global pandemic is upon us. The effects of climate change are getting more and more apparent each day. The guy you like <span class="emphasise">still</span> hasn't answered your text. The superpowers of the world are just one slightly too animated tweet away from pressing the nuclear button.</p> <p>We get it.</p><p>It’s more than one person should—nay, could—handle. Sounds like you need to be... Hibr’d™.</p>
+        </div>
+        <div class="hibr-image">
+          <img src="assets/pod.png">
+        </div>
       </div>
     </section>
-    <!-- dont forget to link the javascript file here! -->
+
+    <main>
+
+      <section class="card-full">
+        <h2 class="yellow">Get Hibr'd™</h2>
+        <div class="card">
+
+
+        <form class="form" action="http://httpbin.org/anything" method="POST">
+          
+          <label for="fullName" class="p-small bold">Full name</label>
+          <input type="text" name="Full Name" id="fullName" required>
+
+          <label for="email" class="p-small bold">E-mail</label>
+          <input type="email" name="E-mail" id="email" required>
+          
+          <div class="form-checkcontainer">
+            <label for=""><p class="bold">What are the reasons you want to be Hibr'd™?</p></label>
```

Comment:
No need to wrap it in `label`, because first of all it's not connected to any `input`, second of all it's more of a form section title. `p` tag is completely enough :) Think about more semantic class naming though (same as with class `yellow`)